### PR TITLE
Some cleanup + better plots for near-global

### LINKED ISSUE #237: Some bugs in the near global example
I think there is something wrong in the fluxes with the near-global example. I have ran it locally after it was not running in PR #137 and the results were a little odd.

The simulation is running but the extrema of SST drift extremely quickly, reaching -35 and +47 degree celsius in a month.
This is the video that is produced by the example that shows that temperature increases quite steeply in the tropics.

https://github.com/user-attachments/assets/94017de1-9751-42a1-ad9b-f71080c6431d

The output is the following:
```julia
[ Info: In-painting ECCO temperature
[ Info: In-painting ECCO salinity
[ Info: Initializing simulation...
[ Info: Iter: 0, time: 0 seconds, Δt: 1.500 minutes, max|u|: (0.00e+00, 0.00e+00, 0.00e+00) m s⁻¹, extrema(T): (31.25, -1.96) ᵒC, wall time: 2.681 minutes
[ Info:     ... simulation initialization complete (6.395 seconds)
[ Info: Executing initial time step...
┌ Warning: Simulation stopped during initialization.
└ @ Oceananigans.Simulations ~/.julia/packages/Oceananigans/s1DfC/src/Simulations/run.jl:129
[ Info:     ... initial time step complete (3.408 minutes).
[ Info: Iter: 4800, time: 5 days, Δt: 1.500 minutes, max|u|: (1.73e+00, 2.26e+00, 1.87e-02) m s⁻¹, extrema(T): (34.14, -10.23) ᵒC, wall time: 40.442 minutes
[ Info: Simulation is stopping after running for 1.289 hours.
[ Info: Simulation time 10 days equals or exceeds stop time 10 days.
[ Info: Iter: 9600, time: 10 days, Δt: 1.500 minutes, max|u|: (2.07e+00, 2.05e+00, 9.71e-03) m s⁻¹, extrema(T): (37.57, -15.24) ᵒC, wall time: 36.879 minutes
[ Info: Initializing simulation...
[ Info:     ... simulation initialization complete (1.970 ms)
[ Info: Executing initial time step...
[ Info:     ... initial time step complete (3.960 seconds).
[ Info: Iter: 10320, time: 15 days, Δt: 10 minutes, max|u|: (1.94e+00, 1.97e+00, 1.61e-02) m s⁻¹, extrema(T): (41.27, -17.67) ᵒC, wall time: 5.780 minutes
[ Info: Iter: 11040, time: 20 days, Δt: 10 minutes, max|u|: (1.79e+00, 1.87e+00, 1.83e-02) m s⁻¹, extrema(T): (42.93, -25.24) ᵒC, wall time: 5.736 minutes
[ Info: Iter: 11760, time: 25 days, Δt: 10 minutes, max|u|: (1.73e+00, 2.56e+00, 1.75e-02) m s⁻¹, extrema(T): (45.31, -31.84) ᵒC, wall time: 5.722 minutes
[ Info: Iter: 12480, time: 30 days, Δt: 10 minutes, max|u|: (2.90e+00, 2.25e+00, 2.19e-02) m s⁻¹, extrema(T): (46.06, -35.93) ᵒC, wall time: 5.718 minutes
```

### PR DESCRIPTION
**Pull request by navidcy**
Some cleanup.

~We can work towards closing #237 as well!~

### CODE REVIEW

File: examples/near_global_ocean_simulation.jl [hunk 1]
```diff
@@ -212,6 +225,9 @@ sn = @lift begin
     parent(un) .= parent(u[$n])
     parent(vn) .= parent(v[$n])
     compute!(s)
+    sn = interior(s)
+    sn[land] .= NaN
+    view(sn, :, :, 1)
```
Comment (glwagner) [inline examples/near_global_ocean_simulation.jl:230]: Why don't we use the Makie extension here?

Comment (navidcy) [reply examples/near_global_ocean_simulation.jl:230]: Makie extension doesn't work for sliced fields (ones that correspond e.g. to `indices = (:, :, 40:40)`); I was about to make a MWE.

Similar for T and e...

Comment (glwagner) [reply examples/near_global_ocean_simulation.jl:230]: ok would be good to have an issue

Comment (glwagner) [reply examples/near_global_ocean_simulation.jl:230]: is this closed with recent PR in oceananignas?

**Comment by navidcy** [comment]
I'd like to merge this because the plots [are much more improved](https://clima.github.io/ClimaOceanDocumentation/previews/PR239/literated/near_global_ocean_simulation/#A-pretty-movie). We can sort the sea-ice model issue in a new PR.

File: examples/near_global_ocean_simulation.jl (hunk 1 above)
Comment (navidcy) [reply examples/near_global_ocean_simulation.jl:230]: https://github.com/CliMA/Oceananigans.jl/issues/3921

**Review by glwagner — APPROVED** [review]

**Pull request merged**